docs/getting-started.md

```diff
@@ -88,6 +89,50 @@ We can use the redis image based on alpine linux from the DockerHub.
 The containerd client uses the `Opts` pattern for many of the method calls.
 We use the `containerd.WithPullUnpack` so that we not only fetch and download the content into containerd's content store but also unpack it into a snapshotter for use as a root filesystem.
 
+Let's put the code together and see it in action.
+
+```go
+package main
+
+import (
+        "context"
+        "log"
+
+        "github.com/containerd/containerd"
+        "github.com/containerd/containerd/namespaces"
+)
+
+func main() {
+        if err := redisExample(); err != nil {
+                log.Fatal(err)
+        }
+}
+
+func redisExample() error {
+        client, err := containerd.New("/run/containerd/containerd.sock")
+        if err != nil {
+                return err
+        }
+        defer client.Close()
+
+        ctx := namespaces.WithNamespace(context.Background(), "example")
+        image, err := client.Pull(ctx, "docker.io/library/redis:alpine", containerd.WithPullUnpack)
+        if err != nil {
+                return err
+        }
+        log.Printf("Successfully pulled %s image\n", image.Name())
+
+        return nil
+}
+```
+
+```bash
+> go build -o example main.go
```

Comment:
@ringanta  What I'm looking for are the steps I'd need to do in order to get this example to run.  When I just copied your example to my system and did  the two commands you had I'd a build failure.  (See below).  So what I'd like to see are whatever steps necessary if you drop the file onto a new machine and you want it to run.  I think that is you first need to clone containerd,  create this file in the main directory of containerd, do export GOPATH=`pwd`, then your build and sudo.  But to be honest, I'm not 100% sure what it takes to get this running as you intended.

So basically whatever steps someone would have to take if they'd not used containerd before and want to get your example running.

$ go build -o example main.go
main.go:4:9: cannot find package "context" in any of:
	/usr/lib/golang/src/pkg/context (from $GOROOT)
	($GOPATH not set)
main.go:7:9: cannot find package "github.com/containerd/containerd" in any of:
	/usr/lib/golang/src/pkg/github.com/containerd/containerd (from $GOROOT)
	($GOPATH not set)
main.go:8:9: cannot find package "github.com/containerd/containerd/namespaces" in any of:
	/usr/lib/golang/src/pkg/github.com/containerd/containerd/namespaces (from $GOROOT)
	($GOPATH not set)